python/dgl/nn/pytorch/conv/gatv2conv.py

```diff
@@ -0,0 +1,291 @@
+"""Torch modules for graph attention networks v2 (GATv2)."""
+# pylint: disable= no-member, arguments-differ, invalid-name
+import torch as th
+from torch import nn
+
+from .... import function as fn
+from ...functional import edge_softmax
+from ....base import DGLError
+from ..utils import Identity
+from ....utils import expand_as_pair
+
+# pylint: enable=W0235
+class GATv2Conv(nn.Module):
+    r"""
+
+    Description
+    -----------
+    Apply GATv2 from
+    `How Attentive are Graph Attention Networks?<https://arxiv.org/pdf/2105.14491.pdf>`__
+    over an input signal.
+
+    .. math::
+        h_i^{(l+1)} = \sum_{j\in \mathcal{N}(i)} \alpha_{i,j} W^{(l)} h_j^{(l)}
+
+    where :math:`\alpha_{ij}` is the attention score bewteen node :math:`i` and
+    node :math:`j`:
+
+    .. math::
+        \alpha_{ij}^{l} &= \mathrm{softmax_i} (e_{ij}^{l})
+
+        e_{ij}^{l} &= \vec{a}^T\mathrm{LeakyReLU}\left(W[h_{i} \| h_{j}]\right)
+
+    Parameters
+    ----------
+    in_feats : int, or pair of ints
+        Input feature size; i.e, the number of dimensions of :math:`h_i^{(l)}`.
```

Comment:
Add "If the layer is to be applied to a unidirectional bipartite graph, ``in_feats`` specifies the input feature size on both the source and destination nodes. If a scalar is given, the source and destination node feature size would take the same value."